CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - master
   - New
+    - New CLI flag `-ic` to ignore comments from wordlist.
```

Comment:
This should be removed from here and added in #138

Reply:
Fixed